pkgs/os-specific/linux/kernel/modinst-arg-list-too-long_5.3.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/scripts/Makefile.modinst b/scripts/Makefile.modinst
+index 07650ee..934a7a8 100644
+--- a/scripts/Makefile.modinst
++++ b/scripts/Makefile.modinst
+@@ -9,3 +9,4 @@ include scripts/Kbuild.include
+ include scripts/Kbuild.include
+
+-modules := $(sort $(shell cat $(if $(KBUILD_EXTMOD),$(KBUILD_EXTMOD)/)modules.order))
++modules := $(sort $(foreach f,$(wildcard $(MODVERDIR)/*.mod),$(shell \
++    grep -h '\.ko$$' '$f')))
```

Comment:
Hm, did you check whether this patch is still needed? Like eg. build the kernel by adding a bunch of environment variables and see whether modules are properly installed.

Reply:
> Hm, did you check whether this patch is still needed? Like eg. build the kernel by adding a bunch of environment variables and see whether modules are properly installed.

As it is now, AFAICT no modules actually get installed :(.  Found out when went to reboot and expected modules weren't found... (somewhat surprised initrd generator didn't die during build or something,but shrug).

Removing this patch (64829de228144c7560d3cfb00629627453a7df40) builds and installs the modules.  I'm not sure what construction was previously running into argument length limits, but via notes here: 98842f84e2dfbb4f60b025cb75120de5afc700af (which links to kernel commit changing `Makefile.modinst`, hence the conflict) it might be possible it's no longer a problem.  Or maybe only shows up on Darwin or when cross-compiling or something O:).

Reply:
@dtzWill: Okay, can you verify if it **really** works by adding some more environment variables to the build?